isis/tests/BundleMeasureTests.cpp

```diff
@@ -0,0 +1,166 @@
+#include "BundleControlPoint.h"
+#include "BundleMeasure.h"
+#include "BundleSettings.h"
+#include "ControlMeasure.h"
+#include "ControlPoint.h"
+#include "IException.h"
+#include "SpecialPixel.h"
+
+#include "gmock/gmock.h"
+
+// To be used for basic setup of a BundleMeasure test object. (Fixture Function)
+class BundleMeasure_CreateObj : public ::testing::Test
+{
+
+  protected:
+    // Create a ControlPoint object using the default constructor
+    // to feed its pointer into the BundleControlPoint constructor.
+    ControlPoint controlPoint;
+
+    // Create a ControlMeasure object using the default constructor
+    // to feed its pointer into the BundleMeasure constructor.
+    ControlMeasure controlMeasure;
+
+    // Since BundleMeasure and BundleControlPoint do not have
+    // default constructors, and C++ does not allow for object declarations
+    // pointers to each respective object are used.
+    BundleMeasure *testBundleMeasurePtr;
+    BundleControlPoint *testBundleControlPointPtr;
+
+
+    void SetUp() override
+    {
```

Comment:
Yo, curly bracket on it's own line? That's not very cash money of you.

Reply:
I have a class that requires brackets to be placed in this manner, so I guess it's a habit from that. I'll fix it to be more in line with how the other gtests do it.